doc/guides/running_simulations.rst

```diff
@@ -199,3 +199,45 @@ a Poisson spike train using different seeds and output files for each run:
         nest.Connect(nrn, sd)
     
         nest.Simulate(100)
+
+Monitoring elapsed time
+-----------------------
+
+The progress of the simulation can be monitored by setting:
+
+::
+
+    SetKernelStatus({"print_time": True})
+
+If enabled, a line is printed to screen at every time step of the simulation to
+track the percentage and the absolute elapsed biological simulation time as
+well as the real-time factor, for example:
+
+::
+
+    [ 25% ] Biological simulation time: 250.0 ms, Real-time factor: 2.6711
+
+The *real-time factor* is defined as the quotient of *wall-clock time* (which
+is also known as real time) and the *biological simulation time* (which is the
+argument to the ``Simulate()`` call):
+
+.. math::
+
+    q_\text{real} = \frac{T_\text{wall}}{T_\text{sim}}
+
+If the real-time factor is larger than `1` as in the example above, the
+simulation runs more slowly than the wall-clock time passes.
+
+In the case that a simulation script contains multiple ``Simulate()`` calls,
```

Comment:
You could shorten this to ``In case a simulation script contains``

Reply:
Ok.